File: swagger/models/stored_card_info.py
from .base_model import BaseModel


class StoredCardInfo(BaseModel):
	"""
	no description available

	Attributes:
	"""

	input_map = {
		'LastFour': 'last_four',
		}

	output_map = {
		'last_four': 'LastFour',
		}

File: swagger/Models/StoredCardInfo.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class StoredCardInfo extends BaseModel
{
	/**
	 * no description available
	 * 
	 * @var string 
	 */
	public $lastFour;

	protected function getInputMap()
	{
		return [
			'LastFour' => ['lastFour', 'string', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'lastFour' => ['LastFour', 'string', null],
			];
	}
}
